code/modules/clothing/head/misc.dm

```diff
@@ -28,6 +28,28 @@
 	item_state = "that"
 	flags = FPRINT
 	siemens_coefficient = 0.9
+	var/magichatchance = 5 //%
+	var/ismagichat
+	var/timer
+
+/obj/item/clothing/head/that/New()
+	if(prob(magichatchance))
+		desc = "It's a magic looking hat."
+		ismagichat = TRUE
+		wizard_garb = TRUE
+
+/obj/item/clothing/head/that/attackby(obj/item/W, mob/user)
+	if(ismagichat)
+		var/allowed_wands = list(/obj/item/item_handle, /obj/item/weapon/cane, /obj/item/weapon/nullrod, /obj/item/weapon/staff)
```

Comment:
make this static

Reply:
Wouldn't it be cleaner to do it this was rather than 4 `is_type(W, XYZ) || etc.` in a single if()?

Reply:
no, but it doesn't really matter either way
using a non-static list with `is_type_in_list` is the worst possible option, though